config/setup/errors.go

```diff
@@ -21,12 +23,17 @@ func Errors(c *Controller) (middleware.Middleware, error) {
 	// Open the log file for writing when the server starts
 	c.Startup = append(c.Startup, func() error {
 		var err error
-		var file *os.File
+		var file io.Writer
 
 		if handler.LogFile == "stdout" {
 			file = os.Stdout
 		} else if handler.LogFile == "stderr" {
 			file = os.Stderr
+		} else if handler.LogFile == "syslog" {
+			file, err = gsyslog.NewLogger(gsyslog.LOG_ERR, "SYSLOG", "caddy")
```

Comment:
From caddy's point of view, the *n* in `LOCALn` is arbitrary. `LOCAL0` works for me.

All *facility* gives you is a string to match in syslog.conf -- to treat lines from different facilities in different ways. So caddy's syslog facility need only be known and documented. Despite suggesting it above, I don't really think log facility needs to be configurable, at least not now in this PR.